### ISSUE TITLE
src_vault onsidered harmful

### ISSUE THREAD

**Issue by chr-engwer** [brainstorm-tools/bst-duneuro#11]
https://github.com/brainstorm-tools/bst-duneuro/blob/c65f7b07d0208c3cec0167e5a4fc508a9742262e/config/setup_bst_duneuro.sh#L45

You alternatively offer the option to unpack a tgz stored in the repo. I think this is dangerous, as it might easily diverge from the state of the repositories. I suggest to simply drop this option and remove the file from the repository.

**Comment by juangpc** [comment]
It was a solution for when GitLab server was down. I think of it as a backup, a vault, that can't hurt.

**Comment by tmedani** [comment]
yes, it was the case in the beginning of the project when GitLab was down. 
maybe a solution is to check, time to time,  the difference with the state of the repo, and update it from time to time with an auto script.

**Comment by juangpc** [comment]
See 558ce41ddd59e801a10e93e16c867687dfa666b3

**Comment by chr-engwer** [comment]
I still consider this binary blob not good practice. First because you have to make sure that information can't diverge and second because storing binary data (and updating it over and over again) is not something git is made for.

**Comment by chr-engwer** [comment]
.. But in the end you need to maintain it :wink:
